scripts/generate_service.go

```diff
@@ -0,0 +1,305 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"go/format"
+	"log"
+	"os"
+	"strings"
+	"text/template"
+)
+
+type ComputeApiGenVersion int
+
+const (
+	none ComputeApiGenVersion = iota
+	v1
+	v0beta
+)
+
+var OrderedComputeApiVersions = []ComputeApiGenVersion{
```

Comment:
I'm not sure of another way to not have to attach explicit values to enum values w/o iota; the ordered list is the source of truth for ordering and the `iota` just conveniently means I don't need to set values.